src/utils/requestModule/shared/duplicate.ts

```diff
@@ -0,0 +1,10 @@
+import * as lodash from 'lodash'
+import { usernameListType } from "utils/shared"
+
+export const duplicateCheck = async(usernamelists: Promise<usernameListType>): Promise<string[]> => {
```

Comment:
I think there is no need to wrap this logic around a function. especially `removeDuplicateData`